test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -48,5 +60,89 @@ test.describe(
 <!-- /wp:navigation -->`
 			);
 		} );
+
+		test( 'default to my only existing menu', async ( {
+			editor,
```

Comment:
We can also bring out `page` here so that we don't have to do `editor.page` below. We should probably mark `editor.page` and similar variables as private class fields.